src/index.js

```diff
@@ -1,4 +1,3 @@
 import renderApplication from './apps/main'
 
 renderApplication(document.getElementById('root'))
```

Comment:
In der .editorconfig steht
`insert_final_newline = true`

d.h. wird beim nächsten mal wieder zurückgechanged. richte dir am besten editorconfig und eslint in deiner IDE ein